tests/Mocks/SampleSubClass.php

```diff
@@ -4,7 +4,7 @@
 
 class SampleSubClass extends SampleClass
 {
-    public function action() : void
+    public function action(): void
```

Comment:
Do changes like this one intentional? or my accident via the IDE?

Reply:
I don't think you have accidentally committed that - seems we are using that return type declaration format consistently via PHPCS